mlir/test/Conversion/MIOpenToGPU/multiple_kernels.mlir

```diff
@@ -0,0 +1,35 @@
+// RUN: mlir-opt -miopen-lowering -miopen-affine-transform -miopen-affix-params -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4 -miopen-lowering-step5 -convert-miopen-to-gpu="kernel-name=step1,step2,step3,step4" | mlir-opt
+// RUN: mlir-opt -miopen-lowering -miopen-affine-transform -miopen-affix-params -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4 -miopen-lowering-step5 -convert-miopen-to-gpu="kernel-name=step1,step2,step3,step4" | FileCheck %s
+
```

Comment:
I think make sense to also test the following scenarios, just to make sure it can fail gracefully:
 - kernel name overflow: `kernel-name=step1,step2,step3,step4,step5`
 - subset of kernel name: `kernel-name=step1`, or `kernel-name=step1,step3`
 - mismatch of kernel name: `kernel-name=step5`

Reply:
solving the issue when the test is run in a multi-threaded environment. will add the amendments after that.

Reply:
resolve the issue and amended the test per review comment